test/kubernetes_cli.go

```diff
@@ -37,6 +40,14 @@ func NewKogitoClientBuilder() *fake.ClientBuilder {
 	return fake.NewClientBuilder().WithScheme(s)
 }
 
+// NewKogitoClientBuilderWithAdditionalScheme creates a new fake.ClientBuilder with the right scheme references
+func NewKogitoClientBuilderWithAdditionalScheme(gv schema.GroupVersion, types ...runtime.Object) *fake.ClientBuilder {
```

Comment:
Is this used anywhere?

Reply:
No, I introduced it for a test that I removed.

Do you think we can leave it there or is it better to remove this utility func as well?

Reply:
Sure, we can leave it here, we may use it for other tests.